packages/angular_devkit/build_angular/src/builders/karma/tests/options/aot_spec.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import { execute } from '../../index';
+import { BASE_OPTIONS, KARMA_BUILDER_INFO, describeKarmaBuilder } from '../setup';
+import { BuilderMode } from '../../schema';
+
+describeKarmaBuilder(execute, KARMA_BUILDER_INFO, (harness, setupTarget) => {
+  describe('Option: "aot"', () => {
+    it('enables aot', async () => {
+      await setupTarget(harness);
+
+      await harness.writeFiles({
+        'src/aot.spec.ts': `
+          import { Component } from '@angular/core';
+          
+          describe('Hello', () => {
+            it('should *not* contain jit instructions', () => {
+              @Component({
+                template: 'Hello',
+              })
+              class Hello {}
+
+              expect((Hello as any).ɵcmp.template.toString()).not.toContain('jit');
+            });
+          });
+`,
+      });
+
+      harness.useTarget('test', {
+        ...BASE_OPTIONS,
+        aot: true,
+        /** Cf. {@link ../builder-mode_spec.ts} */
+        polyfills: ['zone.js', '@angular/localize/init', 'zone.js/testing'],
```

Comment:
Should we factorize this in `BASE_OPTIONS`? or create a `BASE_POLYFILLS`? or an object mother: `optionsMother.base().withPolyfills()`?

Reply:
`describeKarmaBuilder` already runs the builder in both modes automatically. You shouldn't need any override but `aot: true` in here if everything works as expected. The builder mode spec is special because it explicitly tests that the mode can be changed by explicitly setting this. But it seems wasteful to rerun this test case twice in application builder mode (which I think is what's currently happening with this file).